plugins/module_utils/crypto/module_backends/csr.py

```diff
@@ -346,7 +346,7 @@ def generate_csr(self):
             try:
                 csr = csr.add_extension(cryptography.x509.NameConstraints(
                     [cryptography_get_name(name, 'name constraints permitted') for name in self.name_constraints_permitted],
```

Comment:
The same is true here:
```suggestion
                    [cryptography_get_name(name, 'name constraints permitted') for name in self.name_constraints_permitted] if self.name_constraints_permitted else None,
```